primers.go

```diff
@@ -103,3 +104,80 @@ func MeltingTemp(sequence string) float64 {
 	meltingTemp, _, _ := SantaLucia(sequence, primerConcentration, saltConcentration, magnesiumConcentration)
 	return meltingTemp
 }
+
+/******************************************************************************
+Start of the De Bruijn stuff
+******************************************************************************/
+
+// DeBruijn generates a DNA DeBruijn sequence with alphabet ATGC. DeBruijn sequences are basically a string with all unique substrings of an alphabet represented exactly once.
+// https://en.wikipedia.org/wiki/De_Bruijn_sequence
+// https://rosettacode.org/wiki/De_Bruijn_sequences#Go
+// Pulled and adapted from here
```

Comment:
Is there anyway we can just import a function directly from this golang library you mentioned?

https://rosettacode.org/wiki/De_Bruijn_sequences#Go

writing our own means we maintaining and testing our own.

Reply:
Unfortunately, there is not. It is not a Golang library, it is just a repository of ways to do basic algorithms in different languages.